### PR TITLE
changed: Rename ambiguous IsInhibit-builtin to IdleShutdownInhibited

### PR DESCRIPTION
**Pull request by arnova**

### CODE REVIEW

**Review by AlwinEsch — CHANGES_REQUESTED** [review]
Of the changes OK, only if it is used a lot after all, don't remove it immediately and look to the developers to adjust their parts,

Can understand the background of the change, but a small descriptive text in the request would also be good :smirk:.

The necessary change would be here:
https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/DialogButtonMenu.xml#L71-L76
Therefore "Request changes".

File: xbmc/guilib/guiinfo/SystemGUIInfo.cpp [hunk 1]
```diff
@@ -595,7 +595,7 @@ bool CSystemGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
     case SYSTEM_ISSTANDALONE:
       value = g_application.IsStandAlone();
       return true;
-    case SYSTEM_ISINHIBIT:
```
[the remark this answers is not in the document]
Comment (arnova) [reply xbmc/guilib/guiinfo/SystemGUIInfo.cpp:598]: @AlwinEsch : The problem with this is that it requires reserving an additional label but I can also see the advantage in informing users. It's hard to predict the scope of this change but I suspect it's limited.

**Comment by arnova** [comment]
> Of the changes OK, only if it is used a lot after all, don't remove it immediately and look to the developers to adjust their parts,
> 
> Can understand the background of the change, but a small descriptive text in the request would also be good smirk.
> 
> The necessary change would be here: https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/DialogButtonMenu.xml#L71-L76 Therefore "Request changes".

@AlwinEsch : Updated this.

File: xbmc/GUIInfoManager.cpp [hunk 2]
```diff
@@ -1798,7 +1798,7 @@ const infomap system_labels[] = {{"hasnetwork", SYSTEM_ETHERNET_LINK_ACTIVE},
                                  {"batterylevel", SYSTEM_BATTERY_LEVEL},
                                  {"friendlyname", SYSTEM_FRIENDLY_NAME},
                                  {"alarmpos", SYSTEM_ALARM_POS},
-                                 {"isinhibit", SYSTEM_ISINHIBIT},
```
Comment (garbear) [inline xbmc/GUIInfoManager.cpp:1801]: Can you re-add this line as

```c++
{"isinhibit", SYSTEM_IDLE_SHUTDOWN_INHIBITED}, // Deprecated
```

That way, if skins or add-ons don't update, they won't break.

Comment (arnova) [reply xbmc/GUIInfoManager.cpp:1801]: @garbear : Updated

**Review by garbear — APPROVED** [review]

**Comment by arnova** [comment]
@AlwinEsch : You ok with this in its current state?

**Review by AlwinEsch — APPROVED** [review]
Yes is good for me :+1:

**Pull request merged**